documentation/asciidoc/topics/yaml/prometheus_service_monitor.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
```

Comment:
Comments in source is not good case for copy/paste

Reply:
You mean that comments aren't great because you also copy them along with the yaml? 

I realize that might not be ideal but you can always just delete the comments. And, because you copy the comments, you get the field descriptions in your source so don't have to refer back to docs.

Do you feel like callouts are better?

Reply:
Oh, sorry, comments are not a problem for OCP console.